scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/synchronization/SynchronizationWrapper.java

```diff
@@ -0,0 +1,248 @@
+        internalAPI.clear(originator, taskId, channel);
+    }
+
+    @Override
+    public Set<String> keySet(String channel) throws InvalidChannelException {
+        return internalAPI.keySet(originator, taskId, channel);
+    }
+
+    @Override
+    public <T extends Serializable> Collection<T> values(String channel) throws InvalidChannelException {
+        return internalAPI.values(originator, taskId, channel);
+    }
+
+    @Override
+    public <T extends Serializable> Set<Map.Entry<String, T>> entrySet(String channel) throws InvalidChannelException {
+        return internalAPI.entrySet(originator, taskId, channel);
+    }
+
+    @Override
+    public <T extends Serializable> T getOrDefault(String channel, String key, T defaultValue)
+            throws InvalidChannelException {
+        return internalAPI.getOrDefault(originator, taskId, channel, key, defaultValue);
+    }
+
+    @Override
+    public <T extends Serializable> void forEach(String channel, String action)
+            throws InvalidChannelException, CompilationException {
+        internalAPI.forEach(originator, taskId, channel, action);
+    }
+
+    @Override
+    public <T extends Serializable> void replaceAll(String channel, String function)
+            throws InvalidChannelException, CompilationException, IOException {
+        internalAPI.replaceAll(originator, taskId, channel, function);
+    }
+
+    @Override
+    public <T extends Serializable> T putIfAbsent(String channel, String key, T value)
+            throws InvalidChannelException, IOException {
+        return internalAPI.putIfAbsent(originator, taskId, channel, key, value);
+    }
+
+    @Override
+    public <T extends Serializable> boolean remove(String channel, String key, T value)
+            throws InvalidChannelException, IOException {
+        return internalAPI.remove(originator, taskId, channel, key, value);
+    }
+
+    @Override
+    public <T extends Serializable> boolean replace(String channel, String key, T oldValue, T newValue)
+            throws InvalidChannelException, IOException {
+        return internalAPI.replace(originator, taskId, channel, key, oldValue, newValue);
+    }
+
+    @Override
+    public <T extends Serializable> T replace(String channel, String key, T value)
+            throws InvalidChannelException, IOException {
+        return internalAPI.replace(originator, taskId, channel, key, value);
+    }
+
+    @Override
+    public <T extends Serializable> T computeIfAbsent(String channel, String key, String mappingFunction)
+            throws InvalidChannelException, CompilationException, IOException {
+        return internalAPI.computeIfAbsent(originator, taskId, channel, key, mappingFunction);
+    }
+
+    @Override
+    public <T extends Serializable> T computeIfPresent(String channel, String key, String remappingFunction)
+            throws InvalidChannelException, CompilationException, IOException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)